yb-voyager/src/srcdb/ora2pg_extract_schema.go

```diff
@@ -91,5 +91,9 @@ func ora2pgExtractSchema(source *Source, exportDir string) {
 			utils.WaitChannel <- 0 //stop waiting with exit code 0
 			<-utils.WaitChannel
 		}
+		if err := processImportDirectives(utils.GetObjectFilePath(schemaDirPath, exportObject)); err != nil {
+			utils.ErrExit(err.Error())
+		}
```

Comment:
I was thinking that `ExportSchema()` function will be a better place to put this in. Any specific reason to place it here.

Reply:
No specific reason, just noticed the following:
- Making use of the for loop already present which iterates over the different object types
- ora2pgExtractSchema() is common for MySQL and Oracle, no need to enter this code segment twice when it can be factored like this

Reply:
We can think about it like this: it is the responsibility of `SourceDB.ExportSchema()` to abstract the peculiarities of the underlying tool's output format by transforming the output of the tool to the format expected by the rest of the voyager code.